my-first-react-app/src/hello.js

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react';
```

Comment:
The filename of the component should have the same casing as the class that it exports. So this should be `Hello.js`